packages/typespec-test/test/openai/generated/openapi/2023-07-01-preview/openapi.json

```diff
@@ -1335,36 +1311,6 @@
         "b64_json"
       ]
     },
-    "ImageSize": {
```

Comment:
@timotheeguerin Regarding the openapi emitter do we have any fix for versioning and do you have any idea of this change? According to tsp `ImageSize` should be added since version `v2023_06_01_Preview`.

https://github.com/Azure/autorest.typescript/blob/7e7d27973daa4ee6ef081e54c47a0c4f2977153b/packages/typespec-test/test/openai/spec/models/images.tsp#L15-L25

Reply:
should be a bug https://github.com/Azure/typespec-azure/issues/838

Reply:
this was merged you can update to autorest 0.42.1